sdv/cag/fixed_increments.py

```diff
@@ -0,0 +1,218 @@
+"""FixedIncrements CAG pattern."""
+
+import pandas as pd
+
+from sdv._utils import _create_unique_name
+from sdv.cag._errors import PatternNotMetError
+from sdv.cag._utils import (
+    _get_invalid_rows,
+    _remove_columns_from_metadata,
+    _validate_table_and_column_names,
+    _validate_table_name_if_defined,
+)
+from sdv.cag.base import BasePattern
+
+
+class FixedIncrements(BasePattern):
+    """Ensure every value in a column is a multiple of the specified increment.
+
+    Args:
+        column_name (str):
+            Name of the column.
+        increment_value (int):
+            The increment that each value in the column must be a multiple of. Must be greater
+            than 0 and a whole number.
+        table_name (str, optional):
+            The name of the table that contains the column. Optional if the
+            data is only a single table. Defaults to None.
+    """
+
+    @staticmethod
+    def _validate_init_inputs(column_name, increment_value, table_name):
+        if not isinstance(column_name, str):
+            raise ValueError('`column_name` must be a string.')
+        if increment_value <= 0:
```

Comment:
It will crash here if passed a non-numerical value.

Reply:
Good catch. Fixed:
https://github.com/sdv-dev/SDV/pull/2409/commits/ab7486461cdacaa4a32ab55016ba3204f780c406#diff-442749eb6608ea424c56131f8b5c2633ab037f4f758317361cc150224d2d4620R34